app/Domain/Subscriptions/Subscription.php

```diff
@@ -13,9 +13,9 @@ public function __construct(
         public readonly string $name,
         public readonly string $description,
         public readonly Currency $currency,
-        public readonly int $price,
+        public readonly float $price,
         public readonly BillingInterval $billingInterval,
-        public readonly ?int $fee,
+        public readonly ?float $fee,
```

Comment:
As discussed on Slack, Subscription / SubscriptionModel should also have an integration type column so we that when its linked to a new integration, we can filter out the subscriptions that are not meant for the integration's type.

Reply:
Add IntegrationType on Subscription Nova Resource